spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -777,16 +777,27 @@ def download_key(id: nil)
       parse_response(response)
     end
 
-    def create_key!(name: nil, service_configs: nil)
+    def create_key!(name: nil, service_configs: nil, scope: 'team')
       fetch_csrf_token_for_keys
 
+      # Format service configurations for APNS
+      service_configs_requests = (service_configs || []).map do |config|
```

Comment:
For the other [different services](https://www.rubydoc.info/gems/fastlane/2.162.0/Spaceship/Portal/Key), DEVICE_CHECK_ID service will work if you add this to the service config request array: 
 ````
 {
      serviceId: YOUR_HARDCODED_DEVICE_CHECK_ID,  // DQ8HTZ7739
  }
```

havent figured new api shape for `MUSIC_KIT_ID` yet :(

Reply:
```json
{
    "name": "Test Device Check Key",
    "serviceConfigurationsRequests": [
        {
            "isNew": true,
            "serviceId": "DQ8HTZ7739",
            "identifiers": {}
        }
    ],
    "teamId": "9VYG78W35D"
}
```

Reply:
MUSIC_KIT_ID
```json
[
    {
        "isNew": true,
        "serviceId": "6A7HVUVQ3M",
        "identifiers": {
            "music": [
                "4H4P58CJTN"
            ]
        }
    }
]
```